### ISSUE TITLE
Improvements

### ISSUE THREAD

**Issue by jamesli2021** [withastro/studio-issues#17]
Need polished UI, I can find lots of area including font need revamp in Studio.

What does this mean on this page and dev will not know what to do.
"Database is empty."

If you need some example, UpCloud Dashboard is the best I can find.

**Comment by itsMapleLeaf** [comment]
Hi, thanks for the feedback. For this to be actionable, it'd be more helpful to point out specific places you'd like to see improvements, and alternative suggestions. "It needs to be better" is too vague to start with 😅

**Comment by jamesli2021** [comment]
As I have pointed out the message you know in the studio. I think you notice the message?

"Database is empty"

and I have sugged inspiration from UpCloud which you could login and get the idea how their layout, all you need to do is to sign up here:
https://signup.upcloud.com/

**Comment by itsMapleLeaf** [comment]
Yes, I noticed that part, but I was referring mainly to "there are several areas, including the font" bit. What areas? Which font would be better?

Then you suggest to look at this one other specific product for inspiration, without any other specifics beyond that.

The suggested alternative for the "Database is empty message" is locked behind a login wall which starts a 3-day trial. A screenshot, or writing the suggestion directly in text what you think should be there instead, would be appreciated.
